sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/RenderClientImpl.java

```diff
@@ -1,70 +1,24 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+// Licensed under the MIT License. See License.txt in the project root for
```

Comment:
Just for sanity: did you modify this code manually? If not, then you can just resolve this comment here.